### DISCUSSION TITLE
🎉🎉🎉 MobaXterm Portable v24.2 (2024-07-02) 汉化版发布啦 !

### DISCUSSION THREAD

**Discussion by RipplePiam** [RipplePiam/MobaXterm-Chinese-Simplified#71] [General]
**🎉🎉🎉 MobaXterm Portable v24.3 (2024-10-30) 汉化版发布啦 !**

**更新说明：**

- **拿去卖钱的，祝你这辈子吃不上两个菜！**
- 本次更新可解决 #65
- == 以下是官方更新 ==
- 主要是新增了图形化的宏记录等功能
![new_feat](https://github.com/user-attachments/assets/2cf07ef6-aa86-4a09-8777-40ebb85514cd)

```
版本 24.3 (2024-10-30)

新功能：添加了“图形宏”，可以记录您的鼠标和键盘输入并在以后重播
改进：您现在可以将宏放入文件夹中，然后使用拖放功能进行组织
改进：为 VNC 会话添加了 IPv6 支持（使用新的 VNC 引擎）
改进：在专业定制器中添加了新的安全设置，以禁用“列出开放网络端口”工具
改进：在专业定制器中添加了新的安全设置，以禁用“管理员命令提示符”工具
改进：在专业定制器中设置“禁用 X11 自动启动”安全设置时，全局设置窗口中的选项被禁用
改进：在专业定制器中设置“禁用 X11 远程访问”安全设置时，全局设置窗口中的选项被禁用
改进：嵌入到会话中的宏现在默认加密
改进：延迟启动时的一些检查以提高整体性能
改进：通过捕捉比以前更多的键盘事件改进了宏录制
改进：在选项卡上添加了右键单击选项，可以关闭所有非活动选项卡
改进：运行宏后立即将焦点发送到终端
改进：默认将 Bash 提示符中的日期格式从“(day)/(month)/(year)”更改为“(year)-(month)-(day)”
改进：改进了鼠标光标位于会话树上时显示的工具提示
改进：在启动时添加了更多调试信息，特别是针对 Slash 文件夹识别
改进：对于新的 MobaXterm 用户，默认将本地终端切换为 64 位
错误修复：在专业定制器中禁用“重新连接 RDP”功能后，“重新连接”按钮仍然在 RDP 顶部栏中可用
错误修复：宏中的“WaitFor”条件现在可以正确处理预期模式开头和结尾的空格
错误修复：抑制安装过程中弹出控制台窗口闪烁（修复 Ahmed Sherif 报告的 CVE-2024-48200）
错误修复：更改主密码后侧边栏中的宏列表未刷新
```

<hr /><em>This discussion was created from the release <a href='https://github.com/RipplePiam/MobaXterm-Chinese-Simplified/releases/tag/v0.24.3.10'>🎉🎉🎉 MobaXterm Portable v24.2 (2024-07-02) 汉化版发布啦 !</a>.</em>

**Comment by King-Arthur19** [comment]
不知是我电脑还是啥原因，用ssh会话是，有时候输着就输不了，要多按几下能用，要我输入密码就难受，不知道有没有漏了

**Reply by RipplePiam** [reply]
字符输入吗？是否键盘问题？

**Reply by King-Arthur19** [reply]
键盘没问题，我用自己电脑就没啥问题，不管输入什么都没用，只有多按几下才有用

**Comment by binchen4102** [comment]
感谢博主

**Comment by wjx9694** [comment]
为什么终端内右键还是英文

**Comment by lisn84** [comment]
![微信截图_20250318110633](https://github.com/user-attachments/assets/c5c51386-5ad6-40fc-8df3-6bfc018d0872)
老师您好，请问这是什么原因啊？没有注册类

**Comment by juniusxie** [comment]
Thanks you
<img width="911" height="523" alt="MobaXterm" src="https://github.com/user-attachments/assets/dd1dacba-1a87-459e-a08e-4383eeb622e2" />
